packages/frontend/app/components/bulk-new-users.js

```diff
@@ -182,26 +181,29 @@ export default class BulkNewUsersComponent extends Component {
    **/
   async getFileContents(file) {
     this.fileUploadError = false;
+    const existingUsernames = await this.getExistingUserNames();
     return new Promise((resolve) => {
       const allowedFileTypes = ['text/plain', 'text/csv', 'text/tab-separated-values'];
       if (!allowedFileTypes.includes(file.type)) {
-        const intl = this.intl;
-        this.set('fileUploadError', true);
-        throw new Error(intl.t('general.fileTypeError', { fileType: file.type }));
+        this.fileUploadError = true;
+        throw new Error(this.intl.t('general.fileTypeError', { fileType: file.type }));
```

Comment:
this is an improvement over the current implementation (which throws an error as well, but it's not visible to the end user - only to be found in the browser console).

however it's not ideal:
![image](https://github.com/user-attachments/assets/19566753-4bf0-45a0-a71e-0f01344ada66)

Reply:
Interesting filename, that `DOOM PILG..er`. I'm guessing https://boardgamegeek.com/boardgame/373657/doom-pilgrim has been visited recently.

Reply:
https://github.com/ilios/frontend/pull/8254 should help with avoiding such an error from even being generated.